### DISCUSSION TITLE
Text fields are stretching a lot sometimes

### DISCUSSION THREAD

**Discussion by yaroslavyaroslav** [community/community#35578] [Projects and Issues]
Here's a vid. It appears on both iOS and iPadOS.

https://user-images.githubusercontent.com/16612247/194746681-e5375967-e987-4c3a-8ae7-85f8f46c2ddf.MOV

**Comment by dennewbie** [comment]
What if you change the phone's orientation?

**Reply by yaroslavyaroslav** [reply]
Nothing, but it seems like 1.86 beta fixed it.

**Reply by yaroslavyaroslav** [reply]
It's back.
But rotating helps.
https://user-images.githubusercontent.com/16612247/198095764-0886e263-6db6-4143-94c0-002997d71da1.MOV

**Reply by dennewbie** [reply]
Fantastic, it's really better now. Yes, the rotation lets you see a more complete UI. Thank you for keeping me updated. 😊
